samples/tfm/tfm_hello_world/src/main.c

```diff
@@ -82,4 +101,14 @@ void main(void)
 		printk("SHA256 digest:\n");
 		print_hex_number(hello_digest, 32);
 	}
+
+	/* Configure properly the XL1 and XL2 pins so that the low-frequency crystal
+	 * oscillator (LFXO) can be used.
+	 * This configuration has already been done by TF-M so this is redundant.
+	 */
+	printk("Configuring MCU selection for LFXO\n");
```

Comment:
Let's move the printf down one line or else this is going to look strange for nrf91.

Reply:
I did it like this because of the `samples/tfm/tfm_hello_world/sample.yaml`, which expects this output.
I don't know how to make it pass for nrf91 without it.

Reply:
I think that makes it hard to add something below it. Correct me if I'm wrong.

Reply:
I suggest we either remove the sample.yaml regex line entirely, I think it doesn't care about extra trailing printfs.

Or do:

"(Configuring MCU selection for LFXO)?"